Makefile

```diff
@@ -0,0 +1,34 @@
+SHELL := /bin/bash -euo pipefail
+
+export GO111MODULE := on
+export ADDON_TESTS_PER_ADDON_WAIT_DURATION := 10m
+export GIT_TERMINAL_PROMPT := 1
+export ADDON_TESTS_SETUP_WAIT_DURATION := 30m
+export GOPRIVATE := github.com/mesosphere/kubeaddons
+
+
+.DEFAULT_GOAL := test
+
+.PHONY: git-credentials
+git-credentials:
+ifndef GITHUB_TOKEN
+	@echo "GITHUB_TOKEN not specified, ignore git-credentials target"
+else
```

Comment:
Can we add a check here to ensure that a user running "make test" doesn't overwrite their git config?

Reply:
Excellent comment, @joejulian  - I accidentally did that myself.!

Reply:
:heavy_plus_sign:

Reply:
Let's try doing this instead to use ssh_keys https://github.com/mesosphere/kommander-catalog-api/pull/2/files#diff-06ec1ff8ccf12f4fb12bbd2e25d159dfR1